src/unix/linux_like/linux/mod.rs

```diff
@@ -4520,6 +4536,20 @@ pub const NET_DCCP: ::c_int = 20;
 pub const NET_IRDA: ::c_int = 412;
 
 f! {
+    // This is a workaround since glibc provides no wrappers for these syscalls.
+    // Therefore we implement them directly through `syscall`
+    pub fn sched_getattr(
+        pid: ::pid_t,
+        attr: *mut ::sched_attr,
+        size: ::c_uint,
+        flags: ::c_uint
+    ) -> ::c_int {
+        ::syscall(::SYS_sched_getattr, pid, attr, size, flags) as ::c_int
+    }
+    pub fn sched_setattr(pid: ::pid_t, attr: *mut ::sched_attr, flags: ::c_uint) -> ::c_int {
+        ::syscall(::SYS_sched_setattr, pid, attr, flags) as ::c_int
+    }
```

Comment:
The libc crate is a thin wrapper, and if glibc doesn't expose it, it shouldn't be added.

Reply:
Thanks for your response. 
Before opening this PR I had seen that there are already some methods in the android target which are not exposed and use the syscall directly, so I hoped this might also be applicable here.

E.g.
https://github.com/rust-lang/libc/blob/e077dbfe7faec63e7ea315722a596f9c062f0701/src/unix/linux_like/android/b32/x86/mod.rs#L591-L615

Reply:
I understand, thanks again for your time.